src/main/java/racingcar/model/Name.java

```diff
@@ -0,0 +1,28 @@
+package racingcar.model;
+
+public class Name {
+
+    private static final int MAX_NAME_LENGTH = 5;
+    private static final String MAX_NAME_LENGTH_ERROR_MESSAGE = "[ERROR] 이름은 5자 이하여야 합니다. ";
+    private final String name;
```

Comment:
상수와 필드 사이에 빈줄을 추가하면 가독성이 더  좋아질 것 같아요.
```suggestion
    private static final String MAX_NAME_LENGTH_ERROR_MESSAGE = "[ERROR] 이름은 5자 이하여야 합니다. ";
    
    private final String name;
```